src/ontology/cl-edit.owl

```diff
@@ -31101,6 +31104,38 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:36543915") Annotation(ob
 AnnotationAssertion(rdfs:label obo:CL_4033026 "lung perichondrial fibroblast")
 EquivalentClasses(obo:CL_4033026 ObjectIntersectionOf(obo:CL_4033025 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002048)))
 
+# Class: obo:CL_4040000 (glial restricted tripotential precursor cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:10719353") obo:IAO_0000115 obo:CL_4040000 "A glial precursor cell that generates oligodendrocytes and type-1 and type-2 astrocytes. It has been shown in some mammals that this cell type may express A2B5, nestin, FGFR-1, FGFR-2, FGFR-3, PLP, and DM-20 antigens. Unlike oligodendrocyte precursor cell, it does not initially express PDGFR-alpha and can differentiate into both type-1 and type-2 astrocytes.")
```

Comment:
Of note, 'type-1" and "type-2 astrocytes" do not currently appear in CL and seem to be candidates for new terms.

Reply:
This can be requested with a NTR (new term request) issue: https://github.com/obophenotype/cell-ontology/issues/new?assignees=&labels=new+term+request&template=a_adding_term.md&title=%5BNTR%5D

Reply:
@bvarner-ebi OK. I will do this asap (pb not today)